src/scripts/contract_allocations.py

```diff
@@ -24,21 +24,21 @@ def non_wallet_allocation(network: str, allocation_file: File) -> int:
         if contracts[a.Account] is True
     ], key=lambda t: t.Airdrop, reverse=True)
 
-    allocation_total, found = 0, 0
-    for allocation in sorted_contract_allocations:
-        if allocation.Account in not_wallets:
-            gno_allocation = allocation.Airdrop + allocation.GnoOption
+    airdrop_total, found = 0, 0
```

Comment:
Not sure if renaming `allocation` to `airdrop` in this and the following cases since it's about the sum of airdrop plus gno allocation.
I don't have strong feelings against the new name, it's just that it makes it harder to see what actually changed in the diff. (Which seem to be only print statements)